schemas/os.yml

```diff
@@ -24,7 +24,14 @@
       type: keyword
       example: "Mac OS X"
       description: >
-        Operating system name.
+        Operating system name, without the version.
```

Comment:
I think an OS name is always without the version? I see the context you are coming from but I don't think we should mention that here.

Reply:
The user agent parser gives us the following. Check out `os`:

```JSON
          "user_agent" : {
            "patch" : "3538",
            "major" : "70",
            "minor" : "0",
            "os" : "Mac OS X 10.14.0",
            "os_minor" : "14",
            "os_major" : "10",
            "name" : "Chrome",
            "os_name" : "Mac OS X",
            "device" : "Other"
          }
```

I do think that's useful to have this value to compare overall OS usage. The broken down os name and os version in two fields is useful for broader analysis (os name) and then filtering down one OS at a time to see the version breakdowns. But it's not obvious how to use these two fields to get a sense of the overall usage of each specific OS version, all at the same time.

Reply:
Wouldn't it be that there is first a filter on the OS and then an aggregation on the version to get a correct result?

Reply:
If you want to look at the breakdown within one OS, yes, filtering first works, and then the versions you're looking at are consistent and make sense within the context of that OS.

If you want to look at the overall "population" however (e.g. looking at all the OSes accessing your website), then having the full name with version is helpful.

Then you can see a ranking like:

- Mac OS X 10.12: 43345
- Windows 10: 40300
- Mac OS X 10.11: 20000
- Windows 7: 12000

and so on...

Reply:
> I think an OS name is always without the version? I see the context you are coming from but I don't think we should mention that here.

Actually this brings up an interesting case to mind. Do people consider "Windows 7" to be the OS name, or is it "Windows"? I think if we don't keep this clarification, we'll get "Windows 7" and "Windows 10" in this field (just without the minor & patch versions).